src/routes/index.ts

```diff
@@ -0,0 +1,67 @@
+import { Application } from "express";
+import { errorHandler } from "../middlewares/errorHandler.middleware";
+import { notFoundHandler } from "../middlewares/errorHandler.middleware";
+
+class AppRouter {
```

Comment:
A good way, but if you have a lot of routes method init would be enormous and not readable. So I propose creating a health route file, setting up a route by default Express method, and adding it to this class. Also, all of the global middleware, swagger, and error handlers can be added here. If you need help, pls ping me.

Reply:
Great advice, I used to do it this way before, but I wasn't sure if it was appropriate in this case. I'll make the changes and ping you to take a look :)

Reply:
Moved health routes to the api/health.route.ts file .  I made some changes to the code, please take a look when you have time. Thank you for the comments :)